core/arch/arm/include/sm/optee_smc.h

```diff
@@ -422,6 +422,31 @@
 #define OPTEE_SMC_VM_DESTROYED \
 	OPTEE_SMC_FAST_CALL_VAL(OPTEE_SMC_FUNCID_VM_DESTROYED)
 
+/*
+ * Query OP-TEE about number of supported threads
+ *
+ * Normal World OS or Hypervisor issues this call to find out how much threads
+ * OP-TEE support. In other hand, how many standard calls can be issued in
```

Comment:
s/much/many/
s/support/supports/
s/In other hand/A.k.a/ or /That is (how many ...)/